rpm/libblockdev.spec

```diff
@@ -12,13 +12,43 @@ BuildRequires: pkgconfig(udev)
 BuildRequires: pkgconfig(libkmod)
 BuildRequires: gettext-devel
 BuildRequires: autoconf
+BuildRequires: autoconf-archive
 BuildRequires: automake
 BuildRequires: libtool
+BuildRequires: libfdisk-devel
+# i=1; for j in 00*patch; do printf "Patch%04d: %s\n" $i $j; i=$((i+1));done
+Patch0001: 0001-utils-Allow-passing-NULL-for-error.patch
```

Comment:
Can you explain why we need these 30 patches? They seem much more complex than what we had before and also do quite a lot more stuff, so it's a bit difficult to understand how they are a replacement for the patches we used to have.

Reply:
The patches backport the removal of libparted support to use fdisk which we already use.
I applied all patches doing this instead of the part that touches the vfat fs plugin to keep all in line and not have to change them.